src/dss.jl

```diff
@@ -0,0 +1,5774 @@
+module lib
```

Comment:
I'd call this module `Lib`. Modules are normally capitalized.

Reply:
I'd also call this file `Lib.jl` instead of `dss.jl`. It'd make it easier to find.

Reply:
Renamed to `lib.jl` in 2611a31